gateway/engine/beans/pom.xml

```diff
@@ -12,6 +12,10 @@
 
   <dependencies>
     <dependency>
+      <groupId>net.openhft</groupId>
+      <artifactId>zero-allocation-hashing</artifactId>
+    </dependency>
+    <dependency>
```

Comment:
No, I’ll check tomorrow. It’s self-contained so should be simple. Otherwise I can fall back onto something else.

> On 25 Apr 2016, at 19:14, Eric Wittmann <notifications@github.com> wrote:
> 
> In gateway/engine/beans/pom.xml <https://github.com/apiman/apiman/pull/460#discussion_r60960656>:
> 
> > @@ -12,6 +12,10 @@
> >  
> >    <dependencies>
> >      <dependency>
> > +      <groupId>net.openhft</groupId>
> > +      <artifactId>zero-allocation-hashing</artifactId>
> > +    </dependency>
> > +    <dependency>
> I didn't realize there was a new dependency - did you check with Nick to see if this was usable?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/apiman/apiman/pull/460/files/37b99699b175c1b33ab65e353ebef9a3e753604b#r60960656>